tests/AgentTests/FunctionalAPITests.hs

```diff
@@ -728,13 +728,15 @@ testEnablePQEncryption =
     -- switched to smaller envelopes (before reporting PQ encryption enabled)
     sml <- largeMsg g PQSupportOn
     -- fail because of message size
-    Left (A.CMD LARGE _) <- tryError $ A.sendMessage ca bId PQEncOn SMP.noMsgFlags lrg
+    (6, PQEncOff) <- A.sendMessage ca bId PQEncOn SMP.noMsgFlags lrg
+    get ca ##> ("", bId, MERR 6 (A.CMD {cmdErr = LARGE, errContext = "CryptoLargeMsgError"}))
```

Comment:
by encrypting late we now learn of LARGE message asynchronously and command itself returns success. this would lead to creation of unsent chat items, though they should be marked with error (red x) afterwards. maybe we should add more or less safe limit for unencrypted size, or test encryption immediately upon scheduling delivery, though it is wasteful for most cases